sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/CustomProperties.java

```diff
@@ -0,0 +1,133 @@
+package com.microsoft.azure.mobile;
+
+import com.microsoft.azure.mobile.utils.MobileCenterLog;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+/**
+ * Custom properties builder.
+ * Collect multiple properties for send its at once in the same log.
+ */
+public class CustomProperties {
```

Comment:
File is not formatted also, convention is to commit through Android Studio and check all options except check todo and update copyright.